mods/rv/rules/ai-only-units.yaml

```diff
@@ -0,0 +1,229 @@
+### This file defines units can be only built by AI player, which
+### 1. will use diffirent set of logic compared to orignal units used by AI
+### 2. are pre-loaded transprot units with unload logic.
+
+fv-e1:
+	Inherits: fv
+	Buildable:
+		BuildPaletteOrder: 31
+		Prerequisites: ~prerequisite.aiplayer, ~techlevel.1
+	ProductionCostMultiplier:
+		Multiplier: 125
+	Cargo:
+		InitialUnits: e1
+
+fv-ggi:
+	Inherits: fv-e1
+	Buildable:
+		BuildPaletteOrder: 32
+		Prerequisites: ~prerequisite.aiplayer, ~techlevel.1
+	ProductionCostMultiplier:
+		Multiplier: 150
+	Cargo:
+		InitialUnits: ggi
+
+fv-gren:
+	Inherits: fv-e1
+	Buildable:
+		BuildPaletteOrder: 33
+		Prerequisites: ~prerequisite.aiplayer, gaairc, ~techlevel.2
+	ProductionCostMultiplier@1:
+		Multiplier: 150
+	Cargo:
+		InitialUnits: gren
+
+fv-cleg:
+	Inherits: fv-e1
+	Buildable:
+		BuildPaletteOrder: 33
+		Prerequisites: ~prerequisite.aiplayer, gatech, ~techlevel.4
+	ProductionCostMultiplier@1:
+		Multiplier: 150
+	Cargo:
+		InitialUnits: cleg
+	
+
+bfrt-squad-1:
+	Inherits: bfrt
+	Selectable:
+		Class: bfrt
+	Buildable:
+		Prerequisites: ~prerequisite.aiplayer, gatech, ~techlevel.4
+	Cargo:
+		InitialUnits: ggi, ggi, ggi, gren, gren
+	ProductionCostMultiplier@1:
+		Multiplier: 150
+	RenderSprites:
+		Image: bfrt
+	RenderVoxels:
+		Image: bfrt
+
+bfrt-squad-2:
+	Inherits: bfrt-squad-1
+	Cargo:
+		InitialUnits: cleg, cleg, ggi, ggi, cleg
+	ProductionCostMultiplier@1:
+		Multiplier: 200
+
+bfrt-squad-3:
+	Inherits: bfrt-squad-1
+	Buildable:
+		Prerequisites: ~prerequisite.aiplayer, gatech, ~techlevel.4, ~!tany
+	Cargo:
+		InitialUnits: ggi, ggi, ggi, ggi, tany
+	ProductionCostMultiplier@1:
+		Multiplier: 200
+
+^AItransportControl:
+	Inherits@AI: ^AIMicroManagable
+	Cargo:
+		LoadedCondition: loaded
+	AutoDeployer@AI:
+		RequiresCondition: ai-control && loaded
+		DeployChance: 100
+		DeployTicks: 50
+		UndeployTicks: 200
+
+htk-dron:
+	Inherits: htk
+	Inherits@AI: ^AItransportControl
+	Selectable:
+		Class: htk
+	Buildable:
+		Prerequisites: ~prerequisite.aiplayer
+	ProductionCostMultiplier@1:
+		Multiplier: 180
+	Cargo:
+		InitialUnits: dron
+	RenderSprites:
+		Image: htk
+	RenderVoxels:
+		Image: htk
+
+htk-squad-1:
+	Inherits: htk-dron
+	Buildable:
+		Prerequisites: ~prerequisite.aiplayer
+	Cargo:
+		InitialUnits: e2, e2, flakt, flakt, shk
+	ProductionCostMultiplier@1:
+		Multiplier: 300
+
+shad-squad-1:
+	Inherits: shad
+	Inherits@AI: ^AItransportControl
+	Selectable:
+		Class: shad
+	Buildable:
+		Prerequisites: ~prerequisite.aiplayer, ~techlevel.1, ~!LoadedExists1
```

Comment:
Normal BlackHawk requires an Allied Barracks, i think AI ones should too.

Reply:
Ok